### ISSUE TITLE
Separating annotations from registers and stack variables in disasm/LLIL

### ISSUE THREAD

**Issue by amtal** [jonpalmisc/bn-reflection-theme#4]
A minor quibble: in some cases and architectures, annotations can meld surrounding instruction text.

This is noticeable on architectures that (1) shift annotations around via il.Operand(n, expr) so they're mixed in with instruction operands when (2) those operands are are registers or stack offsets and (3) there's no additional `[ ] +0x..` noise to visually break up tokens.

It also applies a bit to user comments, but those stand out already through spacing and the "//" prefix.

### Examples
x86:
![annotationColor](https://user-images.githubusercontent.com/237010/116020465-29b47a00-a5fb-11eb-879b-78c725933246.png)
microblaze:
![annotationColor-ub](https://user-images.githubusercontent.com/237010/116020945-1bb32900-a5fc-11eb-8922-7d80e9452892.png)
### Proposed Fix

Tweak `s/"annotationColor: "blue"/"annotationColor: "purple"/` or maybe `"semi-light"`.

Assuming the current scheme tries to use cool colors for most content but bring important things (function/label addresses) forward with warm colors, this tweak brings annotations forward slightly. 
* Disassembly/LLIL look basically the same, except inline annotations now always stand out.
* User comments stand out significantly more than before.
* `{Register ...}`, and `{Frame offset ...}` noise visible when "Show variable types at top of function" is enabled is brought forward, which I do not like.

I also tried "semi-light", but do not like what happens when you click on a user comment. Making annotations recede makes more sense and works great on variable type definitions, but, well:
![annotation semi-light](https://user-images.githubusercontent.com/237010/116023556-2fad5980-a601-11eb-9391-d19cab40c550.png)

Excellent theme! I really like what it prioritizes, reminds me of my favorite IDA one :)

**Comment by jonpalmisc** [comment]
Hi @amtal! Thanks for submitting this issue. I definitely see what you mean about the disassembly vs. annotation contrast issue, especially on the MicroBlaze architecture. I will spend some time experimenting with different annotation colors to find a solution that offers more contrast and remains visually appealing.

I'd say you can expect an update/fix within a week.

P.S. It's always refreshing seeing a descriptive issue like this that includes examples. Thanks for all of the detail!

**Comment by jonpalmisc** [comment]
That original ETA might have been an overestimate. Take a look at https://github.com/jonpalmisc/bn-reflection-theme/tree/fix-45 and let me know what you think.

**Comment by amtal** [comment]
Yup, that does it. Dunno much about palette design, but stuff's readable for my workflows and looks nice so :+1: :)

Solution to unreadable bytes is prettier than default-Dark, too:
![annotations](https://user-images.githubusercontent.com/237010/116027547-0b09af80-a60a-11eb-8f3e-ff30eff0f817.png)

**Comment by jonpalmisc** [comment]
Glad to hear this helped! Going to leave this issue open for now until it has received more testing and is released. If you come across anything else, feel free to open an issue any time.
